3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -18,18 +18,46 @@ Complete the four functions provided in the starter `index.js` file:
 Try and avoid using global variables. Instead, use function parameters and 
 return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
+async function fetchData(url) {
   // TODO complete this function
+  const response = await fetch(url);
+  const data = await response.json();
+  return data.results;
 }
 
-function fetchAndPopulatePokemons(/* TODO parameter(s) go here */) {
+function fetchAndPopulatePokemons(data) {
   // TODO complete this function
+  const select = document.getElementById('select');
+  data.forEach((pokemon) => {
+    const option = document.createElement('option');
+    select.appendChild(option);
+    option.value = pokemon.name;
+    option.textContent = pokemon.name;
+  });
+  select.addEventListener('change', fetchImage);
 }
 
-function fetchImage(/* TODO parameter(s) go here */) {
+async function fetchImage() {
   // TODO complete this function
+  const response = await fetch(
+    `https://pokeapi.co/api/v2/pokemon/${select.value}`
+  );
+  const data = await response.json();
+  const img = document.createElement('img');
+  img.src = data.sprites.front_default;
+  document.body.appendChild(img);
+  img.style.width = '200px';
```

Comment:
Good work, this code creates a image element every time I select a pokemon. Can you change the code in a way that only one image will be displayed all the time?